### PR TITLE
Feat/user page

### PR DESCRIPTION
**Pull request by romainhk**

### CODE REVIEW

File: Shared/src/consts/projectURLs.ts [hunk 1]
```diff
@@ -70,15 +70,17 @@ export const CENTRALIZATION_ENDPOINTS = {
     STRIPE_CONFIG: '/config',
     STRIPE_PAYMENT_ATTEMPT: '/create-payment-intent',
     STRIPE_CREATE_CUSTOMER: '/create-customer',
```
Comment (Code-Czar) [inline Shared/src/consts/projectURLs.ts:72]: We can remove that as well, not used anymore

File: Shared/src/consts/projectURLs.ts [hunk 2]
```diff
@@ -70,15 +70,17 @@ export const CENTRALIZATION_ENDPOINTS = {
...
-    STRIPE_CHECKOUT_SESSION: '/create_checkout_session'
+    STRIPE_CHECKOUT_SESSION: '/create_checkout_session',
+    STRIPE_CANCEL_SUBSCRIPTION: '/cancel-subscription'
 };
 
 export const CENTRALIZATION_API_URLS = {
     USERS: PROJECT_URLS.CENTRALIZATION_URL + CENTRALIZATION_ENDPOINTS.USERS,
     STRIPE_CONFIG: PROJECT_URLS.CENTRALIZATION_URL + CENTRALIZATION_ENDPOINTS.STRIPE_CONFIG,
     STRIPE_PAYMENT_ATTEMPT: PROJECT_URLS.CENTRALIZATION_URL + CENTRALIZATION_ENDPOINTS.STRIPE_PAYMENT_ATTEMPT,
     STRIPE_CREATE_CUSTOMER: PROJECT_URLS.CENTRALIZATION_URL + CENTRALIZATION_ENDPOINTS.STRIPE_CREATE_CUSTOMER,
```
Comment (Code-Czar) [inline Shared/src/consts/projectURLs.ts:81]: here as well

File: SimpleFrontEnd/desktop/src/pages/Public/UserPage.vue [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+  <q-page padding style="min-width: 800px">
+    <q-tabs v-model="activeTab">
+      <q-tab name="general" :label="generalTabLabel"></q-tab>
+      <q-tab name="subscription" :label="subscriptionTabLabel"></q-tab>
+    </q-tabs>
+
+    <q-tab-panels v-model="activeTab">
+      <!-- General Tab Panel -->
+      <q-tab-panel name="general">
+        <h4 class="text-h6">General Informations</h4><q-list>
```
Comment (Code-Czar) [inline SimpleFrontEnd/desktop/src/pages/Public/UserPage.vue:12]: We should use translation strings later on

File: SimpleFrontEnd/desktop/src/pages/Public/SubscriptionSuccess.vue [hunk 4]
```diff
@@ -0,0 +1,29 @@
+<template>
+  <div>
+    <h1>Subscription Success!</h1>
+    <p>Your subscription is valid. Redirecting to main page...</p>
```
Comment (Code-Czar) [inline SimpleFrontEnd/desktop/src/pages/Public/SubscriptionSuccess.vue:4]: Same as below, translation strings everywhere

File: SimpleFrontEnd/desktop/src/pages/Public/UserPage.vue (hunk 3 above)
Comment (Code-Czar) [reply SimpleFrontEnd/desktop/src/pages/Public/UserPage.vue:12]: simply use {{ $t('<nestedKey>')}}

File: trading-bot-centralization-server/bruno_api.json
Comment (Code-Czar) [inline trading-bot-centralization-server/bruno_api.json:1]: does this file should be added or should stay in your local env ?

**Review by Code-Czar — APPROVED** [review]

File: Shared/src/consts/projectURLs.ts (hunk 1 above)
Comment (romainhk) [reply Shared/src/consts/projectURLs.ts:72]: the create checkout session code in backend ?

File: trading-bot-centralization-server/bruno_api.json
Comment (romainhk) [reply trading-bot-centralization-server/bruno_api.json:1]: its a dev helper file, that should be added imo to help api debug

**Comment by romainhk** [comment]
c'est tout bon pour moi :)

**Pull request merged**